langchain/src/tools/gmail/create_draft.ts

```diff
@@ -0,0 +1,71 @@
+import { google } from "googleapis";
+import { encodeToBase64 } from "@gomomento/sdk-core/dist/src/internal/utils/string.js";
+import { GmailBaseTool, GmailBaseToolParams } from "./base.js";
+
+export interface CreateDraftSchema {
+  message: string;
+  to: string[];
+  subject: string;
+  cc?: string[];
+  bcc?: string[];
+}
+
+export class GmailCreateDraft
+  extends GmailBaseTool
+{
+
+  name = "create_gmail_draft";
+
+  description =
+    "Use this tool to create a draft email with the provided message fields.";
+
+  constructor(fields: GmailBaseToolParams) {
+    super(fields);
+  }
+
+  static args = [
+    { name: "message", description: "The message to include in the draft" },
+    { name: "to", description: "The list of recipients" },
+    { name: "subject", description: "The subject of the message" },
+    { name: "cc", description: "The list of CC recipients", isList: true },
+    { name: "bcc", description: "The list of BCC recipients", isList: true },
+  ];
+
+  protected prepareDraftMessage(message: string,
```

Comment:
why is this protected? did you mean private?

Reply:
I agree, makes more sense to be private.